httsleep/main.py

```diff
@@ -34,6 +34,9 @@ class HttSleeper(object):
                      function that takes the response as an argument returning True.
     :param auth: a (username, password) tuple for HTTP authentication.
     :param headers: a dict of HTTP headers.
+    :param verify: (optional) Either a boolean, in which case it controls whether we verify
+                   the server's TLS certificate, or a string, in which case it must be a path
+                   to a CA bundle to use. Defaults to ``True``.
```

Comment:
It *should* default to `True`, but at the moment it defaults to `None`. Can you fix this please?

Reply:
TL;DR: I copied this description from the `requests` library, which we delegate to; they default to `None` on the API, but then in the depths of the code they set `verify` to the *actual* default of `True` if not already set.

I'm inclined to stick with this description - and code setting! as-is, but happy to discuss further if you like?

References:
-  Description copied from: https://github.com/requests/requests/blob/master/requests/api.py#L39-L41 (or https://github.com/requests/requests/blob/master/requests/sessions.py#L474-L476, it's the same).
-  API defaulting to `None` in the API: https://github.com/requests/requests/blob/master/requests/sessions.py#L444.
-  Where they turn a `None` into `True`: https://github.com/requests/requests/blob/master/requests/sessions.py#L597 (with reference to https://github.com/requests/requests/blob/master/requests/sessions.py#L367-L368).

Reply:
I still feel it'd be more developer-friendly (especially for those using IDEs) if the kwarg matched the documentation. I don't see any downsides to setting the default value to `True` from the get-go.

Reply:
(Sorry, I added that comment weeks ago, but clicked the wrong button on github...)

Reply:
@kopf it's your library 😃  I've changed this as requested in the most recent commit.